openvdb/openvdb/unittest/TestTypes.cc

```diff
@@ -541,8 +541,8 @@ TestTypes::testTypeList()
     static_assert(DulplicateIntTypes::Size == 4, "");
     static_assert(DulplicateRealTypes::Size == 4, "");
     static_assert(DulplicateIntTypes::Index<Int16> == 1, "");
-    static_assert(std::is_same<DulplicateIntTypes::Unique, TypeList<Int32, Int16, Int64>>::value, "");
-    static_assert(std::is_same<DulplicateRealTypes::Unique, TypeList<float>>::value, "");
+    static_assert(std::is_same<DulplicateIntTypes::Unique<>, TypeList<Int32, Int16, Int64>>::value, "");
```

Comment:
While you're touching this file, maybe fix these typos too -> Dulplicate -> Duplicate ?